plugins/coturn/coturn

```diff
@@ -0,0 +1,107 @@
+# -*- sh -*-
+
+set -e
+
+: << =cut
+
+=head1 NAME
+
+coturn - Simple plugin to monitor sessions on coturn server
+
+=head1 CONFIGURATION
+
+This plugin requires a password set in /etc/turnserver.conf for telnet 
+cli access. A telnet client has to be installed. Optionally you can 
+specify the port of the cli interface.
+
+[coturn]
+env.clipassword VERYSECUREPASSWORD
+env.port PORT
+
+=head1 AUTHOR
+
+Copyright (C) 2020 Sebastian L. <sebastiab@momou.ch>
+
+=head1 LICENSE
+
+GPLv2
+
+=head1 MAGIC MARKERS
+
+ #%# family=manual
+ #%# capabilities=autoconf
+
+=cut
+
+. "$MUNIN_LIBDIR/plugins/plugin.sh"
+
+if [ -z ${port+x} ]; then cliport="5766"; else cliport=$port; fi
+
+get_coturn_stats() {
+	echo "open 127.0.0.1 $cliport"
+	sleep 1
+	echo $clipassword
+	sleep 1
+	for i in udp tcp dtls tls
+	do
+		echo "pu $i"
+		sleep 1
+	done
+}
+
+case $1 in
+    autoconf)
+        if [[ -x /usr/bin/turnserver ]]; then
+            echo yes
```

Comment:
Since `telnet` is not such a common tool anymore, I would suggest to also check for its existence (`[ -z "$(which telnet)" ]`).